examples/platforms/cc2538/uart.c

```diff
@@ -54,14 +54,22 @@ static void processTransmit(void);
 static const uint8_t *sTransmitBuffer = NULL;
 static uint16_t sTransmitLength = 0;
 
+typedef struct RecvBuffer
+{
+    uint16_t Offset;
+    uint16_t Length;
+} RecvBuffer;
+
 static uint8_t sReceiveBuffer[kReceiveBufferSize];
```

Comment:
No specific reason.  The buffer needs to be as large as the maximum latency between calls to `processReceive()`.